src/components/ec-filter-popover/ec-filter-popover.story.js

```diff
@@ -0,0 +1,115 @@
+import { text } from '@storybook/addon-knobs';
+import EcFilterPopover from './ec-filter-popover.vue';
+import EcCheckbox from '../ec-checkbox/ec-checkbox.vue';
+
+export default { title: 'Filter Popover' };
+
+export const Primary = () => ({
+  components: { EcFilterPopover, EcCheckbox },
+  data() {
+    return {
+      itemListOne: [{
+        name: 'Test Bank Name',
+        checkbox: null,
```

Comment:
This prop should be called `selected` or `selectedValue`.

Reply:
is this correct or you meant `name`?

Reply:
It should be 
```
itemListOne: [{
        name: 'Test Bank Name',
        selected: false,
}, {
```

so then in the template is used like this:
```
<ec-checkbox
            v-model="item.selected"
```

You can also use isSelected or selectedValue as a name of the prop, it's up to you.